lib/wcc/contentful/store/postgres_store.rb

```diff
@@ -120,13 +119,27 @@ def self.ensure_schema(conn)
         );
         CREATE INDEX IF NOT EXISTS contentful_raw_value_type ON contentful_raw ((data->'sys'->>'type'));
         CREATE INDEX IF NOT EXISTS contentful_raw_value_content_type ON contentful_raw ((data->'sys'->'contentType'->'sys'->>'id'));
+
+        DROP FUNCTION IF EXISTS "upsert_entry";
```

Comment:
do we specify the required minimum postgres version anywhere?

Reply:
No - I did notice that this doesn't work on postgres 9.  How can we specify that?  I know how to specify the 'pg' gem version but not the postgres DB version.